lib/client/api.go

```diff
@@ -116,9 +116,23 @@ type Config struct {
 	// port setting via -p flag, otherwise '0' is passed which means "use server default"
 	HostPort int
 
-	// ProxyHostPort is a host or IP of the proxy (with optional ":https_port,ssh_port,kube_port").
-	// The value is taken from the --proxy flag and can look like --proxy=host:5025,5080
-	ProxyHostPort string
+	// ProxySSHHost is host or IP of the SSH proxy. It is derived from either
+	// the --proxy flag or from the value the proxy advertises in ssh_public_addr.
+	ProxySSHHost string
```

Comment:
I would like to move back to HostPort notation instead of Host and Port, as it's more consistent with kubernetes and is more compact.